examples/hcloud/flake.nix

```diff
@@ -41,7 +41,7 @@
             location = "hel1";
           };
 
-          system.stateVersion = "23.11";
```

Comment:
sorry, i think we should revert this too

Reply:
done!